apiml-static-registration.yaml.template

```diff
@@ -1,11 +1,11 @@
 services:
-  - serviceId: explorer-uss
-    title: USS Explorer
-    description: IBM z/OS Unix Files UI service
-    catalogUiTileId:
-    instanceBaseUrls:
-      - https://${ZOWE_EXPLORER_HOST}:${USS_EXPLORER_UI_PORT}/
-    homePageRelativeUrl:
-    routedServices:
-      - gatewayUrl: ui/v1
-        serviceRelativeUrl: {{ configs.baseUri }} # Note - this needs reading from manifest.yaml/component configs
\ No newline at end of file
+- serviceId: explorer-uss
+  title: USS Explorer
+  description: IBM z/OS Unix Files UI service
+  catalogUiTileId:
+  instanceBaseUrls:
+  - https://${ZOWE_EXPLORER_HOST}:${USS_EXPLORER_UI_PORT}/
+  homePageRelativeUrl:
+  routedServices:
+  - gatewayUrl: ui/v1
+    serviceRelativeUrl: /ui/v1/explorer-uss
```

Comment:
I'm slightly concerned that we have this is 3 places now. If it's hardcoded here I'm not sure if it is needed in both manifest.yaml and package.json?

Reply:
I found this is a little complicated to fix. We will also need a YAML parser to support parsing manifest in order to properly prepare FVT test environment. Right now we don't have a proper template, so are using the embedded environment variables as template format. I will come back to this later to solve this issue.

Reply:
https://github.com/zowe/zlux/issues/567 created.